src/config/dynamic-theme-fixes.config

```diff
@@ -10109,6 +10109,16 @@ INVERT
 
 ================================
 
+*.mitx.*
```

Comment:
```suggestion
mitx.*
```

Reply:
The site is https://lms.mitx.mit.edu

Reply:
Dark reader already by default match all subdomains.